### ISSUE TITLE
Backup fails

### ISSUE THREAD

**Issue by Mebus** [jtpio/trello-full-backup#14]
I get this error:

```
==== Backup initiated
('Backing up to:', '/home/mebus/ZweiteCryptdisk/_BACKUP/Trello')
('Incremental:', False)
('Tokenize:', False)
('Backup closed board:', False)
('Backup archived lists:', False)
('Backup archived cards:', False)
('Attachment size limit (bytes):', 100000000)
==== 
()
('Saving full json for board', u'Arbeit, kplx', 'with id', u'781ca87c2a8510f0f8di945c', 'to', 'Arbeit, kplx_full.json')
Traceback (most recent call last):
  File "/usr/bin/trello-full-backup", line 11, in <module>
    load_entry_point('trello-full-backup==0.2.3', 'console_scripts', 'trello-full-backup')()
  File "/usr/lib/python2.7/site-packages/trello_full_backup/__init__.py", line 5, in main
    cli()
  File "/usr/lib/python2.7/site-packages/trello_full_backup/backup.py", line 302, in cli
    backup_board(board, args)
  File "/usr/lib/python2.7/site-packages/trello_full_backup/backup.py", line 158, in backup_board
    write_file(file_name, board_details)
  File "/usr/lib/python2.7/site-packages/trello_full_backup/backup.py", line 51, in write_file
    with open(file_name, 'w', encoding='utf-8') as f:
TypeError: 'encoding' is an invalid keyword argument for this function
```

Mebus

**Comment by jtpio** [comment]
You seem to be using Python 2.7 (`/usr/lib/python2.7`), which is not supported.
Try to update to Python 3.4+ or use the Docker image.

**Comment by Mebus** [comment]
Ah, works with Python 3.5. 

But I wonder why I can install it via pip from Python2.7.

Mebus

**Comment by jtpio** [comment]
Should be fixed with 26304a5c525534a5cfd4f30599a9f129ace4b6da.
